DopplerCustomDomain/CustomDomainProvider/DomainConfiguration.cs

```diff
@@ -3,5 +3,6 @@ namespace DopplerCustomDomain.CustomDomainProvider
     public class DomainConfiguration
     {
         public string service { get; set; } = string.Empty;
+        public string ruleType { get; set; } = string.Empty;
```

Comment:
What do you think about directly using the enum here in place of string?

It would simplify the controller logic.